### PR TITLE
fixes #23020 - Docker to Container Images

### PR DESCRIPTION
**Pull request by thomasmckay**
Changing the presentation layer (UI) from _Docker_ to _Container Images_. No change to routes and underlying data.

### CODE REVIEW

**Comment by thomasmckay** [comment]
[test katello]

**Comment by sjha4** [comment]
@thomasmckay , This looks good.
However, I do have an [open PR](https://github.com/Katello/katello/pull/7258) somewhat related to this. It's a broken link on the Repository page for Docker Tag which lands on the manifest page. Do you suggest we merge these?

**Comment by thomasmckay** [comment]
@sjha4 Since they are separate issues it is best to keep them in their own PRs. If there are merge conflicts later the losing PR author will need to rebase. ;)

**Comment by thomasmckay** [comment]
@akofink "container" is a running thing, "container image" is the thing that we have in repos. the foreman-docker runs containers, katello handles container images. they really are two separate things.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/docker-tags.routes.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ angular.module('Bastion.docker-tags').config(['$stateProvider', function ($state
         controller: 'DockerTagDetailsController',
         templateUrl: 'docker-tags/details/views/docker-tag-details.html',
         ncyBreadcrumb: {
-            label: "{{ 'Docker Tags' | translate }}",
+            label: "{{ 'Container Tags' | translate }}",
```
Comment (akofink) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/docker-tags.routes.js]: What about here, @thomasmckay?

Comment (thomasmckay) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/docker-tags.routes.js]: thanks!

**Comment by thomasmckay** [comment]
versions for content view docker hub 2
@parthaa @akofink - Should I rename everything from "Docker" to "Container Image"? I think yes but someone ack my sanity. Thanks!

File: app/models/katello/content_view_filter.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@ class ContentViewFilter < Katello::Model
     ERRATA          = Erratum::CONTENT_TYPE
     DEB             = Deb::CONTENT_TYPE
     CONTENT_TYPES   = [RPM, PACKAGE_GROUP, ERRATA, DOCKER, DEB].freeze
-    CONTENT_OPTIONS = { _('Packages') => RPM, _('Package Groups') => PACKAGE_GROUP, _('Errata') => ERRATA, _('Docker') => DOCKER, _('deb Packages') => DEB }.freeze
+    CONTENT_OPTIONS = { _('Packages') => RPM, _('Package Groups') => PACKAGE_GROUP, _('Errata') => ERRATA, _('Container Images') => DOCKER, _('deb Packages') => DEB }.freeze
```
[the remark this answers is not in the document]
Comment (thomasmckay) [reply app/models/katello/content_view_filter.rb]: still docker in pulp

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-docker-repositories.html [hunk 3]
```diff
@@ -92,18 +92,18 @@ <h3 translate>
           <td bst-table-cell>
             <div>
               <span translate>
```
[the remark this answers is not in the document]
Comment (thomasmckay) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-docker-repositories.html]: Keeping all files and even routes unchanged. Thinking ahead a bit I'm not sure if we'll be treating docker images differently from open container initiative (OCI) images. That may adjust my thinking of route names and such. Seems safest just to leave underlying structure unchanged and just change presentation.

**Pull request merged**